dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -1783,6 +1796,7 @@ RtpsUdpDataLink::RtpsReader::process_heartbeat_i(const RTPS::HeartBeatSubmessage
                                                  bool directed,
                                                  MetaSubmessageVec& meta_submessages)
 {
+  // TODO: Delay responses by heartbeat_response_delay_.
```

Comment:
TODO

Reply:
This will be long term addition to RTPS.